ttyplot.c

```diff
@@ -289,9 +290,99 @@ void redraw_screen(const char * errstr) {
     refresh();
 }
 
+// Handle a single value from the input stream.
+// Return whether we got a full data record.
+int handle_value(double value)
```

Comment:
We have stdbool already include, let's return `bool` for this please.

Reply:
Oh, I didn't notice that include, and I was still in the mood of keeping with the “traditional” C style. OK, changed.

Reply:
@edgar-bonet cool, thank you! :+1: